doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -257,20 +264,21 @@ Some operations imply that only a partial representation of an entity will be pr
 
 * `id` and `type` are not allowed in update operations, as they are inmutable properties.
 
-* In the requests in which entity `type` is allowed, it may be omitted. When omitted, the default string value `none`
-  is used for the type.
+* In the requests in which entity `type` is allowed, it may be omitted. When omitted in entity creation operations,
+  the default string value `none` is used for the type.
 
 * In some cases, not all the attributes of the entity are shown, e.g. a query which selects a subset of entity attributes.
 
 * Attribute/metadata `value` may be omitted in requests, meaning that the attribute/metadata has `null` value. It is always present in responses.
 
-* Attribute/metadata `type` may be ommitted in requests. When omitted, the default string value `none`.
+* Attribute/metadata `type` may be ommitted in requests. When omitted in attribute/metadata creation or update operations, the default
+  string value `none` is used for the type.
 
```

Comment:
To be discussed in a next PR. 

NTC for this PR with regard this (eventual future modifications would be done in a next PR).